pkg/admission/validator.go

```diff
@@ -0,0 +1,19 @@
+package admission
+
+import (
+	sav1 "sigs.k8s.io/service-apis/apis/v1alpha1"
+)
+
+func ValidateHTTPRoute(httpRoute sav1.HTTPRoute) (bool, string, error) {
```

Comment:
should we add warnings to the signature? Since k8s webhooks now support warnings.

Reply:
I can do that, looks like this was introduced in 1.19. Since the first implementations will likely be ready around 1.21 release, do you think it is safe to add them now or should I just add a TODO?